lib/stat_tracker.rb

```diff
@@ -104,27 +104,27 @@ def count_of_teams
   end
 
   def best_offense
-    @game_manager.best_offense
+    @game_teams_manager.best_offense
```

Comment:
I changed all of these when I was debugging and I was wrong :(. I think the consensus was to make all the references `@game_team_manager`. That was my bad.